data/augment.py

```diff
@@ -0,0 +1,758 @@
+"""
+
+This data recipe lets the user to augment new features to the dataset using the Augment Cloud Service.
```

Comment:
I would add more description about this recipe from the perspective of DAI.  Example starting with requirements.
SnowFlake,
DataSet
DAI

description about augmentation, where the output of augment will be persisted, and how it is consumed by DAI.

Reply:
Sure, will do it